### PR TITLE
add test function to  check relaxdriver

### PR DESCRIPTION
**Pull request by kzqureshi**

### CODE REVIEW

File: micromagnetictests/calculatortests/relaxdriver.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import discretisedfield as df
+import micromagneticmodel as mm
+import numpy as np
+import pytest
+
+
+def test_relaxdriver(calculator):
+    name = "relaxdriver"
```
Comment (swapneelap) [inline micromagnetictests/calculatortests/relaxdriver.py:8]: Why have you introduced the `name` variable?

File: micromagnetictests/calculatortests/relaxdriver.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    system = mm.System(name=name)
+    system.energy = mm.Exchange(A=A) + mm.Zeeman(H=H)
+    system.m = df.Field(mesh, nvdim=3, value=(0, 1, 0), norm=Ms)
+
+    md = calculator.RelaxDriver()
+    md.drive(system)
+
+    value = system.m(mesh.region.random_point())
+    assert np.linalg.norm(np.subtract(value, (0, 0, Ms))) < 1e-2
```
Comment (swapneelap) [inline micromagnetictests/calculatortests/relaxdriver.py:26]: How did you come up with the value $\texttt{1e-2}$ as the difference of relaxed state with the `z` saturated state?

File: micromagnetictests/calculatortests/relaxdriver.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+def test_check_for_energy_and_dynamics(calculator):
+    system = mm.examples.macrospin()
+    system.energy = 0
+    md = calculator.RelaxDriver()
+
+    with pytest.raises(RuntimeError, match="System's energy is not defined"):
+        md.drive(system)
+
+    system.dynamics = 0
```
Comment (swapneelap) [inline micromagnetictests/calculatortests/relaxdriver.py:41]: Why does relax driver need a dynamics term? I was under the impression that it is the damping term from LLG, and hence, the dynamics is predefined in a way (something similar to the `MinDriver`).

**Review by swapneelap — COMMENTED** [review]
I have added a few questions regarding the changes. Further, do you think we will need to update the tests (maybe `conftest.py` maybe) in `oommfc` as the relax driver is not implemented(?) there?

File: micromagnetictests/calculatortests/relaxdriver.py (hunk 3 above)
Comment (kzqureshi) [reply micromagnetictests/calculatortests/relaxdriver.py]: addressed in ubermag/mumax3c#82

**Pull request merged**